### ISSUE TITLE
windows preference does not display when clicked.

### ISSUE THREAD

**Issue by iamutaki** [TanmayPatil105/verse#2]
![image](https://github.com/user-attachments/assets/7838b921-f50f-42cb-bb7f-b9ced082324b)

I am using Manjaro Linux and installed it, but nothing appears when I try to access the settings in the preferences.

**Comment by TanmayPatil105** [comment]
Hi @cacing69 , Thanks for reporting!
Are you using flatpak or building it locally?
And what version are you using of verse?

**Comment by mr-anji** [comment]
> Hi @cacing69 , Thanks for reporting! Are you using flatpak or building it locally? And what version are you using of verse?

Hello, I'm also on Manjaro (Latest version, updated. running XFCE), and the menu for the preferences doesnt pop up when clicked, I installed it through Flatpak.

**Comment by mr-anji** [comment]
And its version 0.1.3

**Comment by mr-anji** [comment]
also I launched verse through the terminal and when I clicked preferences it reutrned this message
```
$ flatpak run io.github.TanmayPatil105.verse
Traceback (most recent call last):
  File "/app/share/verse/verse/main.py", line 64, in on_preferences_action
    self.preferences_window = VersePreferences()
                              ^^^^^^^^^^^^^^^^^^
  File "/app/share/verse/verse/views/verse_preferences.py", line 47, in __init__
    self.update_widgets()
  File "/app/share/verse/verse/views/verse_preferences.py", line 101, in update_widgets
    secrets = retrieve_secrets()
              ^^^^^^^^^^^^^^^^^^
  File "/app/share/verse/verse/lib/secrets.py", line 49, in retrieve_secrets
    password = Secret.password_lookup_sync(SECRET_SCHEMA, {}, None)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)


```

**Comment by TanmayPatil105** [comment]
Same issue is reported here: https://github.com/flathub/flathub/issues/4441.

I checked if the APPLICATION IDs don't match and I don't see a problem:
https://github.com/TanmayPatil105/verse/blob/main/src/main.py#L34
https://github.com/TanmayPatil105/verse/blob/main/src/lib/secrets.py#L25
https://github.com/TanmayPatil105/verse/blob/main/io.github.TanmayPatil105.verse.json#L2

**Comment by TanmayPatil105** [comment]
I was able to reproduce this on x11
```console
$ flatpak run io.github.TanmayPatil105.verse
Traceback (most recent call last):
  File "/app/share/verse/verse/main.py", line 64, in on_preferences_action
    self.preferences_window = VersePreferences()
                              ^^^^^^^^^^^^^^^^^^
  File "/app/share/verse/verse/views/verse_preferences.py", line 47, in __init__
    self.update_widgets()
  File "/app/share/verse/verse/views/verse_preferences.py", line 101, in update_widgets
    secrets = retrieve_secrets()
              ^^^^^^^^^^^^^^^^^^
  File "/app/share/verse/verse/lib/secrets.py", line 49, in retrieve_secrets
    password = Secret.password_lookup_sync(SECRET_SCHEMA, {}, None)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
```

**Comment by TanmayPatil105** [comment]
Moving this to https://github.com/flathub/io.github.TanmayPatil105.verse/issues

Edit:
I don't think I have permissions for it :)
Can you please open the issue there?

**Comment by mr-anji** [comment]
> Moving this to https://github.com/flathub/io.github.TanmayPatil105.verse/issues
> 
> Edit: I don't think I have permissions for it :) Can you please open the issue there?

definitely!

**Comment by mr-anji** [comment]
> > Moving this to https://github.com/flathub/io.github.TanmayPatil105.verse/issues
> > Edit: I don't think I have permissions for it :) Can you please open the issue there?
> 
> definitely!

https://github.com/flathub/io.github.TanmayPatil105.verse/issues/2

**Comment by TanmayPatil105** [comment]
Hey @cacing69 @mr-anji,
https://github.com/TanmayPatil105/verse/releases/tag/v0.1.5 Should fix it!

**Comment by iamutaki** [comment]
Im using flatpak

**Comment by TanmayPatil105** [comment]
@cacing69 Should be fixed now!
Update it to v0.1.5

**Comment by iamutaki** [comment]
> @cacing69 Should be fixed now! Update it to v0.1.5

its ok, i will check it

**Comment by iamutaki** [comment]
![image](https://github.com/user-attachments/assets/763f5069-7179-41ba-b6af-e47c6b2229aa)
its showed
